protocol-designer/src/steplist/fieldLevel/processing.ts

```diff
@@ -34,21 +29,18 @@ export const trimDecimals = (
  **********************/
 // NOTE: these are often preceded by a Value Caster when composed via composeMaskers
 // in practice they will always take parameters of one type (e.g. `(value: string)`)
-// For the sake of simplicity and flow happiness, they are equipped to deal with parameters of type `mixed`
-
-export const onlyPositiveNumbers: ValueMasker = (value: mixed) =>
+// For the sake of simplicity and flow happiness, they are equipped to deal with parameters of type `unknown`
```

Comment:
If this comment even still applies
```suggestion
// For the sake of simplicity and TS happiness, they are equipped to deal with parameters of type `unknown`
```